src/main/java/com/box/sdk/BoxFile.java

```diff
@@ -516,6 +524,11 @@ public void getRepresentationContent(String representationHint, String assetPath
                 String repContentURLString = null;
                 while (repContentURLString == null) {
                     repContentURLString = this.pollRepInfo(representation.getInfo().getUrl());
+                    try {
+                        Thread.sleep(100);
```

Comment:
why this sleep ?

Reply:
This is basically infinite loop that is doing as many requests as it can. Only thing that can slow it down is rate limiter.  This seems like a least we could improve.